src/gui/qgsfilewidget.cpp

```diff
@@ -51,6 +51,9 @@ QgsFileWidget::QgsFileWidget( QWidget *parent )
   mLinkLabel->setSizePolicy( QSizePolicy::Expanding, QSizePolicy::Preferred );
   mLinkLabel->setTextFormat( Qt::RichText );
   mLinkLabel->hide(); // do not show by default
+  mLinkEditButton = new QToolButton( this );
```

Comment:
To confirm -- this is only shown when links are shown, right? not in the default line edit mode?

Reply:
only shown with the label (links shown) and when not in read only.

Reply:
thanks!